src/main/scala/com/codacy/client/bitbucket/client/RequestResponse.scala

```diff
@@ -8,14 +8,21 @@ sealed trait RequestResponse[+A] {
 
```

Comment:
Deprecate or remove "map"  method, mentioning that don't use this with pagination